test/strategy/DebugConfigStrategyTest.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var common = require('../common');
+var assert = common.assert;
+
+var strategy = require('../../lib/strategy');
+var DebugConfigStrategy = strategy.DebugConfigStrategy;
+
+function MockLogger(listener) {
+  this.listener = listener;
+}
+
+MockLogger.prototype.debug = function (message) {
+  this.listener('debug', message);
+};
+
+describe('DebugConfigStrategy', function () {
+  var mockLogger;
+  var testSection;
+  var testConfig;
+  var testStrategy;
+  var testListener;
+  var loggedTestMessages = [];
+
+  before(function () {
+    testSection = 'test';
+    testConfig = { abc: '123' };
+
+    testListener = function onMessageLogged(type, message) {
+      loggedTestMessages.push({ type: type, message: message });
+    };
+
+    mockLogger = new MockLogger(testListener);
+    testStrategy = new DebugConfigStrategy(mockLogger, testSection);
+  });
+
+  describe('when processing', function () {
```

Comment:
Change this description to `.process(config, callback)` as that's the method that is being tested.